### DISCUSSION TITLE
posts/tutorial3/

### DISCUSSION THREAD

**Discussion by giscus** [7w1/7w1.github.io#5] [Announcements]
# posts/tutorial3/

Welcome back!

https://7w1.github.io/posts/tutorial3/

**Comment by miosad** [comment]
in version 1.13.0 the "radioisotope thermal generator" research is called "pressurised water reactor". wanted to clear it up cause i got confused for a second
![screenshot of research tab](https://imgur.com/a/LhT9ojg)

**Reply by miosad** [reply]
don't know how to edit comments but i need to fix the image source

![screenshot of research tab](https://i.imgur.com/G7Pzgpb.png)

**Reply by miosad** [reply]
nevermind, i see now that they're different

**Comment by dawnfreelance** [comment]
Hello, i am on version 2.51, and there is no option to research cargo bay - small or fuel storage - large. i followed the tutorial to a tee (kinda, i have 16 research and not 8 for some reason)

**Reply by 7w1** [reply]
This tutorial is a bit outdated, some of the upgrade paths have changed in the most recent updates and no longer need to be researched.
